src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -58,8 +79,18 @@ def connect(self, port):
 
         self._connect(port, baudrate=38400)
```

Comment:
You are not changing this, but ideally this baudrate should also come from the config.